ui/main/src/app/modules/cards/components/detail/detail.component.html

```diff
@@ -135,39 +135,6 @@
                 (click)="confirmDeleteCard()" translate>button.delete</button>
             </div>
           </ng-template>
-
-          <ng-template #cardDeletedWithNoErrorPopup let-modal>
-            <div class="modal-header text-center">
-              <span class="modal-title w-100" translate> userCard.deleteCard.cardDeletedWithNoError </span>
-              <div id="opfab-card-details-delete-btn-close" class="opfab-close-modal-icon" aria-label="Close"
-                   (click)="modal.dismiss('Cross click')">
-                <span aria-hidden="true">&times;</span>
-              </div>
-            </div>
-            <div class="modal-body text-center">
-              <button id="opfab-card-details-delete-btn-ok" type="button" class="opfab-btn" (click)="modal.dismiss()"
-                translate>button.ok</button>
-            </div>
-          </ng-template>
-
-          <ng-template #impossibleToDeleteCardPopup let-modal>
-            <div class="modal-header text-center">
-              <span class="modal-title w-100" translate> userCard.deleteCard.error.title </span>
-              <div id="opfab-card-details-delete-error-btn-close" class="opfab-close-modal-icon" aria-label="Close"
-                   (click)="modal.dismiss('Cross click')">
-                <span aria-hidden="true">&times;</span>
-              </div>
-            </div>
-            <div class="modal-body text-center">
-              <p translate> userCard.deleteCard.error.impossibleToDeleteCard </p>
-
-              <button id="opfab-card-details-delete-error-btn-close" type="button" class="opfab-btn"
-                (click)="modal.dismiss()" translate>button.ok</button>
-            </div>
-          </ng-template>
-        </div>
```

Comment:
I think this "< / div>" shouldn't be deleted (line 168). If you are ok, can you modify please ?
Thank you.

Reply:
Done